tpython/lib/utils/futils.py

```diff
@@ -1,136 +1,129 @@
-from genericpath import exists
+from genericpath import isdir
```

Comment:
Not a huge deal here but just so you know there's an accepted practice for ordering imports. Take a look here: https://www.python.org/dev/peps/pep-0008/#imports

Basically its:

Standard library imports
Third-party imports 
Local application imports
Blank line between each of these groups then too